numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -1372,7 +1421,7 @@ PyArray_Correlate2(PyObject *op1, PyObject *op2, int mode)
  * Numeric.correlate(a1,a2,mode)
  */
 NPY_NO_EXPORT PyObject *
-PyArray_Correlate(PyObject *op1, PyObject *op2, int mode)
+PyArray_Correlate(PyObject *op1, PyObject *op2, int mode, npy_intp minlag, npy_intp maxlag, npy_intp lagstep)
```

Comment:
This breaks backwards compatibility for the exported C API. The signature should remain
```PyObject* PyArray_Correlate(PyObject* op1, PyObject* op2, int mode)```